src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IIntellisenseData.cs

```diff
@@ -44,6 +45,8 @@ internal interface IIntellisenseData
         /// </summary>
         public string Script { get; }
 
+        public CultureInfo Locale { get; }
```

Comment:
> public CultureInfo Locale { get; } [](http://example.com/codeflow?start=7&length=35)

nit - can we add  comment about this. 
We have 4 different Cultures, so we should comment about how this one is used.  

"This is the locale for messages like function and parameter help."

(not parser locale. Not error locale. Not runtime locale)